test/e2e/workflow/deployment_test.go

```diff
@@ -198,3 +256,33 @@ func ignoreInitialKubeMacPoolRestart() {
 	By("Ignoring initial KubeMacPool restart")
 	time.Sleep(10 * time.Second)
 }
+
+func retrieveRange() (string, string) {
```

Comment:
I have a bad feeling about peeking into config map as a part of functional tests here. In any case, please move this to check.go as a helper function for CheckUnicast.